packages/audioplayers/example/lib/tabs/controls.dart

```diff
@@ -96,7 +96,7 @@ class _ControlsTabState extends State<ControlsTab>
         ),
         WrappedListTile(
           leading: const Text('Rate'),
-          children: [0.0, 0.5, 1.0, 2.0].map((it) {
+          children: [0.5, 1.0, 2.0].map((it) {
```

Comment:
I think it would be better to use the same example as audioplayers.
https://github.com/bluefireteam/audioplayers/blob/main/packages/audioplayers/example/lib/tabs/controls.dart#L104
If this value is limited, I think it should be passed as an error through the platform code.
(There is no need to change it unless there are problems running or building the example, or unless it is platform-dependent code.)